generator/client/msw-handlers.ts

```diff
@@ -60,12 +58,15 @@ export function generateMSWHandlers(spec: OpenAPIV3.Document) {
      *
      * @see https://mswjs.io/docs/basics/response-transformer#custom-transformer
      */
-    export function json<B>(
+    export function json<B extends DefaultBodyType>(
       body: B,
       options: { status?: number; delay?: number } = {}
-    ): ResponseTransformer<B> {
-      const { status = 200, delay = 0 } = options
-      return compose(context.status(status), context.json(body), context.delay(delay))
+    ): StrictResponse<B> {
+      const { status = 200, delay = 0 } = options;
+      if (delay) {
+        doDelay(delay);
+      }
+      return HttpResponse.json(body, { status });
```

Comment:
This is not bad at all. I think their delay function is async now, which means this would need to be `async` and callers would need to `await` every `json()` call. How about we take `delay` out of this function altogether so we can keep it sync and make callers do `await delay(1000)` directly?

Reply:
Yeah, that sounds good.

Reply:
heh with that change.... I don't think it needs to exist, does it? it basically has the same signature as `HttpResponse.json`